src/Swarm/TUI/Controller/ControllerUtils.hs

```diff
@@ -0,0 +1,24 @@
+{-# LANGUAGE PatternSynonyms #-}
+
+module Swarm.TUI.Controller.ControllerUtils where
```

Comment:
Why not just name this `Swarm.TUI.Controller.Utils`?

Reply:
> Why not `Controller.Utils` and `View.Utils`? I'm not sure what the benefit is of repeating words as in `Controller.ControllerUtils`.

In previous projects that I've worked in, having multiple modules of the same name throughout the hierarchy caused issues for tooling.  But VS Code and our Swarm development workflow doesn't seem to have an issue with it, so I'm happy to name them all `Utils`.